WalletWasabi.Fluent/ViewModels/Navigation/RoutableViewModel.cs

```diff
@@ -72,7 +72,7 @@ public INavigationStack<RoutableViewModel> Navigate()
 			return Navigate(currentTarget);
 		}
 
-		public static INavigationStack<RoutableViewModel> Navigate(NavigationTarget currentTarget)
+		public INavigationStack<RoutableViewModel> Navigate(NavigationTarget currentTarget)
```

Comment:
why this change?

Reply:
Compiler didn't allow this https://github.com/zkSNACKs/WalletWasabi/blob/08192461af32b8b222cc393e30234d64327dda35/WalletWasabi.Fluent/ViewModels/Wallets/Home/History/HistoryItemViewModel.cs#L35

Reply:
RoutableViewModel.Navigate()?

Reply:
fixed.